js/src/personal/maliha/Menu.tsx

```diff
@@ -1,15 +1,34 @@
-import { Burger } from '@mantine/core'
-import { useDisclosure } from '@mantine/hooks'
+/** Component to show menu items */
+import { Text } from '@mantine/core'
+import styles from './Maliha.module.css'
 
-export function Menu() {
-  const [opened, { toggle }] = useDisclosure()
+export function MyMenu() {
   return (
-    <Burger
-      opened={opened}
-      onClick={toggle}
-      aria-label="Toggle navigation"
-      size={50}
-      color="#662100"
-    />
+    <div className={styles.itemWrapper}>
+      <li>
```

Comment:
Does it make sense to use the List and List.Item component in Mantine?
https://mantine.dev/core/list/

Also generally li's should only be in ul or ol.